rails6/en/chapter06-improve-json.adoc

```diff
@@ -741,11 +731,11 @@ And now we can build some tests:
 class ProductTest < ActiveSupport::TestCase
   # ...
   test "should filter products by name" do
-    assert_equal 2, Product.filter_by_title('TV').count
+    assert_equal 2, Product.filter_by_title('tv').count
   end
 
   test 'should filter products by name and sort them' do
-    assert_equal [products(:another_tv), products(:one)], Product.filter_by_title('TV').sort
+    assert_equal [products(:another_tv), products(:one)], Product.filter_by_title('tv').sort
```

Comment:
The lower case is intentional because this test that `Product#filter_by_title` is not case sensitive.

Reply:
But I think this is ok. I will merge as is ;)

Reply:
Great! It's because in the products.yml is in uppercase